### PR TITLE
[#1383] 🆕 Ability to disable database evolutions on a datasource basis

### LINKED ISSUE #1383: Ability to disable database evolutions on a datasource basis
It would be really convenient to be able to disable database evolutions on a datasource basis.
The configuration currently support a global `evolutions.enabled=false` parameter.
Such a parameter could be present per datasource: `db.datasourcename.evolutions.enabled=false`

### PR DESCRIPTION
**Pull request by holajsh**
## Fixes

Fixes #1383 

## Purpose

Add support for a new configuration parameter to allow disabling database evolutions per database.

## References

#1383

### CODE REVIEW

File: framework/src/play/db/Evolutions.java [hunk 1]
```diff
@@ -392,6 +392,10 @@ private boolean isDisabled() {
         return "false".equals(Play.configuration.getProperty("evolutions.enabled", "true"));
     }
 
+    private static boolean isDatabaseEvolutionDisabled(String dbName) {
```
Comment (asolntsev) [inline framework/src/play/db/Evolutions.java:395]: You are doing double-negotiation here: "false" -> "Disabled" -> "!Disabled". The code may be slightly simplified.

Comment (holajsh) [reply framework/src/play/db/Evolutions.java:395]: Of course, in fact I just kept the existing "style" to stay coherent with other methods :D
Can be changed if really required ;)

Comment (asolntsev) [reply framework/src/play/db/Evolutions.java:395]: I am not sure. Probably not such important. :)

**Review by asolntsev — APPROVED** [review]

**Review by xael-fry — CHANGES_REQUESTED** [review]
@holajsh can you update the documentation https://github.com/playframework/play1/blob/master/documentation/manual/evolutions.textile ?
Thanks

**Comment by holajsh** [comment]
> @holajsh can you update the documentation https://github.com/playframework/play1/blob/master/documentation/manual/evolutions.textile ? Thanks

Sure, I have pushed something but I'm not really sure how to check it will display properly...

**Review by xael-fry — APPROVED** [review]

**Pull request merged**